### PR TITLE
Define test_install_uninstall.py

### PR DESCRIPTION
**Pull request by Junefen**
**Description**

- [x] test_install_uninstall_by_yum
- [x] test_install_uninstall_by_rpm

**Test Result**
```
# pytest --disable-warnings -v tests/package/test_install_uninstall.py
======================== test session starts ========================
platform linux -- Python 3.9.9, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /root/workspace/virtwho-test, configfile: pytest.ini
collected 2 items                                                                                                                                

tests/package/test_install_uninstall.py::TestInstallUninstall::test_install_uninstall_by_yum PASSED   [ 50%]
tests/package/test_install_uninstall.py::TestInstallUninstall::test_install_uninstall_by_rpm PASSED   [100%]
```

### CODE REVIEW

**Comment by Junefen** [comment]
Please don't merge it, need updating.

**Comment by Junefen** [comment]
@hkx303  It's ready to review.

**Comment by hkx303** [comment]
This PR deponds on #72

**Comment by hkx303** [comment]
I suggest pushing all local changes for a goal, all changes besides the case file also need to push to the PR. It would be better for the Github workflow. We need to make the upstream work anytime. and it would be easy and safe to rebase one node if we need to recover some changes

File: tests/package/test_install_uninstall.py [hunk 1]
```diff
@@ -5,23 +5,60 @@
...
+        assert package_check(ssh_host, 'virt-who') is False
+        package_install(ssh_host, 'virt-who')
+        assert package_check(ssh_host, 'virt-who') == VIRTWHO_PKG
+
+    @pytest.mark.tier1
+    def test_install_uninstall_by_rpm(self, ssh_host):
+        """Test virt-who install/uninstall by rpm
+
+        :title: virt-who: package: install/uninstall by rpm
+        :id: 4f784cd4-ad46-4457-a20f-7cf07ba44d9e
```
Comment (hkx303) [inline tests/package/test_install_uninstall.py:41]: Same uuid with test_install_uninstall_by_yum

**Review by hkx303 — CHANGES_REQUESTED** [review]
pls generate a new uuid for the case

File: tests/package/test_install_uninstall.py (hunk 1 above)
Comment (Junefen) [reply tests/package/test_install_uninstall.py]: Changed it.

**Review by hkx303 — APPROVED** [review]
ACK

**Pull request merged**